sdk/containerregistry/container-registry/samples-dev/anonymousListTags.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Illustrates how to list the tags for an image in a registry that enables anonymous pull access.
+ * @azsdk-weight 5
+ *
+ */
+
+// A common use case for Azure Container Registries is to view the repositories, artifacts, or tags
+// in a public registry that belongs to someone else. In this case, the user would need to access
+// the registry anonymously. Anonymous access allows a user to list all the collections there, but
+// they wouldn't also have permissions to modify or delete any of the images in the registry.
+import { ContainerRegistryClient } from "@azure/container-registry";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+async function main() {
+  // Get the service endpoint from the environment
+  const endpoint = process.env.CONTAINER_REGISTRY_ENDPOINT || "<endpoint>";
+
+  // Create a new ContainerRegistryClient for anonymous access
+  const client = new ContainerRegistryClient(endpoint);
+
+  // Obtain a RegistryArtifact object to get access to image operations
+  const image = client.getArtifact("library/hello-world", "latest");
+
+  // List the set of tags on the hello_world image tagged as "latest"
+  const tags = image.listTagProperties();
```

Comment:
Same nit as above: I think it's clearer if we inline this into the `for await` statement